Tests/DeckOfPlayingCardsTests/DeckTest.swift

```diff
@@ -1,19 +1,21 @@
-/*
- This source file is part of the Swift.org open source project
- 
- Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
- Licensed under Apache License v2.0 with Runtime Library Exception
- 
- See http://swift.org/LICENSE.txt for license information
- See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-*/
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the example package deckofplayingcard open source project
+//
+// Copyright (c) 2015-2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
 
-import XCTest
+import Testing
 
 import PlayingCard
 import DeckOfPlayingCards
 
-class DeckTests: XCTestCase {
+struct DeckTests {
+    @Test
     func testStandard52CardDeck() {
```

Comment:
I recommend renaming the functions which are gaining the `@Test` attribute to strip the "test" prefix from them. For example, this one could become
```suggestion
    func standard52CardDeck() {
```

(This comment applies broadly to any modernization PR.)

Reply:
Applying that across all these PRs, thank you